truffle.js

```diff
@@ -11,7 +11,13 @@ module.exports = {
   },
   compilers: {
     solc: {
-      version: '0.4.24'
+      version: '0.4.24',
+      settings: {
+        optimizer: {
+          enabled: true,
+          runs: 200
```

Comment:
200 looks like default, when it's enabled. Think that's good, or should we choose a higher number?